src/stdlib_experimental_error.md

```diff
@@ -0,0 +1,106 @@
+# Catching and handling errors
+
+* [`check` - Checks the value of a logical condition](#check---checks-the-value-of-a-logical-condition)
+* [`error_stop` - aborts a program](#error_stop---aborts-a-program)
+
+
+## `check` - Checks the value of a logical condition
+
+### Description
+
+Checks the value of a logical condition.
+
+### Syntax
+
+`call check(condition, msg, code, warn)`
+
+### Arguments
+
+`condition`: Shall be a scalar of type `logical`.
+
+`msg` (optional): Shall be a character expression containing the message to be printed to `stderr`. The default `msg` is 'Check failed'.
```

Comment:
The default `msg` is 'Check failed.'. (there's a period at the end of the string.)